test/common/v8.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const assert = require('assert');
+
+function checkGCProfile(data) {
+  assert.ok(data.version > 0);
+  assert.ok(data.startTime >= 0);
+  assert.ok(data.endtTime >= 0);
```

Comment:
```suggestion
  assert.ok(data.endTime >= 0);
```
Related to the previous review.